ivy.xml

```diff
@@ -49,6 +49,8 @@
     <dependency org="log4j" name="log4j" rev="1.2.16" transitive="false" conf="default"/>
     <dependency org="jline" name="jline" rev="0.9.94" transitive="false" conf="default"/>
 
+    <dependency org="com.google.code.findbugs" name="annotations" rev="3.0.1" conf="default"/>
```

Comment:
I'd prefer to exclude `AuthFastLeaderElection.java` as this is deprecated rather than introducing google library dependency. Does this sound good to you. Thanks!